### PR TITLE
Sort games

### LINKED ISSUE #475: Filters on Games page

### PR DESCRIPTION
**Pull request by EgoriusRus**

### CODE REVIEW

File: components/GamesTable.vue [hunk 1]
```diff
@@ -38,6 +38,11 @@ export default {
       type: Number,
       required: false,
       default: 50
+    },
+    sort: {
```
Comment (Klimonov) [inline components/GamesTable.vue]: bad naming, does not reflect the essence of the content

File: pages/games.vue [hunk 2]
```diff
@@ -29,7 +70,27 @@ export default {
   },
   components: {
     GamesTable,
-    AntHivePageHeader
+    AntHivePageHeader,
+    AntHiveIcon
+  },
+  data() {
+    return {
+      gamesSortPreset: 'best-score',
```
Comment (Klimonov) [inline pages/games.vue]: why you use 'preset' ? is this a common word for sorting type ?
maybe gamesSortType is better.
what you think ?

File: pages/games.vue [hunk 3]
```diff
@@ -29,7 +70,27 @@ export default {
...
+          preset: 'best-score',
+          label: 'Best score'
+        },
+        {
+          preset: 'longest',
+          label: 'Longest games'
+        },
+        {
+          preset: '',
+          label: 'Recent games'
```
Comment (Klimonov) [inline pages/games.vue]: dont forget about i18n

File: lang/en.json [hunk 4]
```diff
@@ -64,7 +64,10 @@
     "hoursAgo": "h ago",
     "minutesAgo": "m ago",
     "secondsAgo": "s ago",
-    "justNow": "Just now"
+    "justNow": "Just now",
+    "bestScore": "Best score",
+    "longest": "Longest games",
+    "recent": "Recent games"
```
Comment (Klimonov) [inline lang/en.json]: name the key as similar to the value
`longestGames`
`recentGames`

**Review by kezlya — APPROVED** [review]
LGTM

**Review by Klimonov — APPROVED** [review]
LGTM

**Pull request merged**